### ISSUE TITLE
Tray icon "Process hacker icon (static)" graphic showing inconsistently.

### ISSUE THREAD

**Issue by nodomw** [winsiderss/systeminformer#1304]
### Brief description of your issue

The tray icon's graphic does not always show as the title says. For a glimpse, it shows a black box with nothing inside of it, presumably a graph.

Before this, I was toying around with icons what I believe happened, was that it either tried to register a graph to the static icon module or vice-versa. 

I am able to reproduce this issue on both my laptop and desktop.

### Steps to reproduce (optional)

1. Cold start the application
2. Enable any tray icon except for "Process hacker icon (static)" first
3. Enable the specific tray icon discussed before

### Environment (optional)

```shell
#Process Hacker
Version: 3.0.4953 (37e0466)

OBJECT INFORMATION
PhObjectTypeObject: 18 objects
PhStringType: 9879 objects
PhBytesType: 2553 objects
PhListType: 350 objects
PhPointerListType: 138 objects
PhHashtableType: 25 objects
PhFileStreamType: 0 objects
PhSymbolProviderType: 0 objects
PhProcessItemType: 285 objects
PhServiceItemType: 826 objects
PhNetworkItemType: 0 objects
PhModuleProviderType: 0 objects
PhModuleItemType: 0 objects
PhThreadProviderType: 0 objects
PhThreadItemType: 0 objects
PhHandleProviderType: 0 objects
PhHandleItemType: 0 objects
PhMemoryItemType: 0 objects
PhImageListItemType: 462 objects

#Windows
Windows 11 Pro (Home on laptop) Build 10.0.22000
```

**Comment by jxy-s** [comment]
Thanks @nodomw for the report. I am able to reproduce this after fiddling around with the tray icons. The static icon appears to not be drawn correctly in certain conditions. I'll investigate further.

**Comment by dmex** [comment]
This was fixed with commit 36012632911414258066015d8cb178cece068522 and the new icon. There just hasn't been a new nightly build with those changes yet.
